__tests__/unit/utils/sanitizeAndTrimText.test.ts

```diff
@@ -0,0 +1,39 @@
+import {
+  sanitizeText,
+  removeExtraWhiteSpaces,
+  sanitizeAndTrimText,
+} from '../../../app/utils/formInputSanitization';
+
+jest.mock('../../../app/utils/formInputSanitization', () => ({
+  ...jest.requireActual('../../../app/utils/formInputSanitization'), // Use actual implementations where not specifically mocked
+  sanitizeText: jest.fn(),
+  removeExtraWhiteSpaces: jest.fn(),
+}));
+
+describe('sanitizeAndTrimText', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const cases = [
+    {
+      input: 'Hello,   Linkta!@#$   ',
+      sanitized: 'Hello,   Linkta!   ',
+      trimmed: 'Hello, Linkta',
+    },
+    {
+      input: 'Test!!! &&& Linkta***  ',
+      sanitized: 'Test &&& Linkta  ',
```

Comment:
Expecting `'Test    &&& Linkta     '`, with 4 spaces after 'Test' instead of 1 and 5 spaces after 'Linkta' instead of 2.